platform-sdk/swirlds-merkledb/src/test/java/com/swirlds/merkledb/collections/HashListByteBufferTest.java

```diff
@@ -36,11 +44,23 @@ class HashListByteBufferTest {
     private static final int LARGE_HASHES_PER_BUFFER = 10_000;
 
     public HashList createHashList(final int numHashesPerBuffer, final long maxHashes, final boolean offHeap) {
-        return new HashListByteBuffer(numHashesPerBuffer, maxHashes, offHeap);
+        final Configuration config = ConfigurationBuilder.create()
```

Comment:
Maybe it would be good to rename test var names as well (similar to `HashListByteBuffer`: `maxHashes` -> `capacity`, etc.)

Reply:
OK, I renamed maxHashes to capacity and numHashesPerBuffer to hashesPerBuffer.